keras/layers/merge.py

```diff
@@ -219,6 +219,9 @@ def compute_mask(self, inputs, mask=None):
     return backend.all(
         backend.concatenate(masks, axis=0), axis=0, keepdims=False)
 
+  def get_config(self):
+    return super(_Merge, self).get_config()
```

Comment:
Just `super()` in Python

Reply:
Updated. My first time knowing this for so many years python usage. :vampire: